lua/nvim-treesitter/highlight.lua

```diff
@@ -68,6 +68,10 @@ hlmap["text.title"] = "TSTitle"
 hlmap["text.literal"] = "TSLiteral"
 hlmap["text.uri"] = "TSURI"
 
+-- Tags
+hlmap["tag"] = "TSTag"
+hlmap["tag.delimiter"] = "TSTagDelimiter"
```

Comment:
You need to declare these in https://github.com/stsewd/nvim-treesitter/blob/master/plugin/nvim-treesitter.vim